src/map/time_server.cpp

```diff
@@ -51,6 +51,8 @@ int32 time_server(time_point tick, CTaskMgr::CTask* PTask)
             conquest::UpdateWeekConquest();
             lastConquestTally = tick;
         }
+
+        roeutils::CycleWeeklyRecords();
```

Comment:
Should this be moved into the block above? I think this will get hit several times.

Reply:
Thanks for the feedback!  The 1h offset threw me originally, but now I understand its purpose.